mapfishapp/src/main/webapp/app/addons/bangeocoder/js/main.js

```diff
@@ -0,0 +1,200 @@
+Ext.namespace("GEOR.Addons");
+
+GEOR.Addons.BANGeocoder = Ext.extend(GEOR.Addons.Base, {
+    win: null,
+    addressField: null,
+    layer: null,
+
+    /**
+     * Method: init
+     *
+     * Parameters:
+     * record - {Ext.data.record} a record with the addon parameters
+     */
+    init: function(record) {
+
+        // create layer : container of geometry to display adress selected
+        this.layer = new OpenLayers.Layer.Vector("__georchestra_bangeocoder", {
+            displayInLayerSwitcher: false,
+            styleMap: new OpenLayers.StyleMap({
+                "default": {
+                	graphicWidth: 20,
+                    graphicHeight: 32,
+                    graphicYOffset: -28, // shift graphic up 28 pixels
+                    //Change the location of picture if not find
+                    externalGraphic : 'app/css/images/pwrs/geoPin.png',
```

Comment:
you should rather use 
```js
externalGraphic: GEOR.config.PATHNAME + "/app/css/images/pwrs/geoPin.png"
```
or else the path will be broken when using a permalink